src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/caseupdate/model/request/ExistingCaseDetails.java

```diff
@@ -5,16 +5,22 @@
 import java.util.Map;
 
 public class ExistingCaseDetails {
+
+    @JsonProperty("id")
+    public final String id;
```

Comment:
will sample-app accept this extra field? Perhaps we should add it on the other side first?

Reply:
sample app changes:
https://github.com/hmcts/bulk-scan-ccd-event-handler-sample-app/pull/113